package.json

```diff
@@ -74,6 +74,8 @@
     "prettier": "^1.16.4",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-hooks-testing-library": "^0.5.0",
+    "react-test-renderer": "^16.8.6",
```

Comment:
Is this dep needed? I'm not seeing it used.

Reply:
It's a peer dependency of `react-hooks-testing-library`.  We use it to render the test component without requiring a DOM to be in place, but it should be the corresponding version to the provided `react` version (also a peer dependency).